source/connect/mongoclient.txt

```diff
@@ -0,0 +1,131 @@
+.. _kotlin-sync-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and performs 
+  operations on the MongoDB deployment.
+
+You can also use ``MongoClientSettings`` to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` object
+to connect to MongoDB.
+
+.. _kotlin-sync-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. TODO, add this as last sentence for ``username:password`` description once a kotlin auth page is made: 
+.. For more information about the ``authSource`` connection option, see :ref:`kotlin-sync-auth`. 
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`kotlin-sync-connection-options` for a full description of
+       these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string?tck=docs_driver_java_sync>` in the
```

Comment:
```suggestion
:manual:`Connection Strings </reference/connection-string?tck=docs_driver_java_sync>` in the
```
Is the ``tck`` correct for the java sync language in the selector? @mcmorisi

Reply:
Java Sync is probably the closest match but it's still technically a different language (despite sharing a lot of syntax with Kotlin). I see that there's no option in the dropdown for kotlin, so I think we can remove the `?tck=docs_driver_java_sync`